pages/glossary/browsing-context-related-nodes.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Set of browsing context related nodes
+key: browsing-context-related-nodes
+---
+
+Two HTML [nodes](https://dom.spec.whatwg.org/#concept-node) are _browsing context related_ if:
+
+- their [shadow including roots](https://dom.spec.whatwg.org/#concept-shadow-including-root) are [fully active](https://html.spec.whatwg.org/#fully-active); and
+- the [browsing contexts](https://html.spec.whatwg.org/#browsing-context) of their [shadow including roots](https://dom.spec.whatwg.org/#concept-shadow-including-root) either have a common [ancestor browsing context](https://html.spec.whatwg.org/#ancestor-browsing-context), or one is [ancestor browsing context](https://html.spec.whatwg.org/#ancestor-browsing-context) of the other, or are the same.
+
+A _set of browsing context related nodes_ is the set of all [nodes](https://dom.spec.whatwg.org/#concept-node) that are browsing context related with a [node](https://dom.spec.whatwg.org/#concept-node) whose [shadow including root]'s [browsing context](https://html.spec.whatwg.org/#browsing-context) is a [top-level browsing context](https://html.spec.whatwg.org/#top-level-browsing-context).
+
+A _set of browsing context related documents_ is the set of all [documents](https://dom.spec.whatwg.org/#concept-document) that are browsing context related with a [document](https://dom.spec.whatwg.org/#concept-document) whose [browsing context](https://html.spec.whatwg.org/#browsing-context) is a [top-level browsing context](https://html.spec.whatwg.org/#top-level-browsing-context).
+
+**Note**: By construction, a set of browsing context related nodes is always maximal for inclusion.
```

Comment:
Why do we need this note?

Reply:
Not strictly needed, but I felt it was a useful clarification. A set of BCR nodes could also be defined as "a set of nodes that are pairwise BCR" (it's an equivalence anyway) in which case it would not necessarily be maximal for inclusion.

In a more math-y text, I would totally define "a set of foo" as the non-maximal for inclusion one, and use (possibly with definition) "a maximal set of foo" for the one that has to be maximal for inclusion. It did hurt a bit my math-self to use "a set of BCR nodes" for what should rather be "a maximal set of BCR nodes". Since we probably won't need the former, it was simpler not to use it. But then, I had to add the clarifying note to point out that these sets are maximal even if the name does not suggest so.